.github/workflows/test.yml

```diff
@@ -164,21 +166,22 @@ jobs:
             mkdir -p ${CI_DIST_IMAGES_OUTPUT}
             cp ${CI_IMAGE_CACHE}/${CI_IMAGE_PR_TAR}.gz ${CI_DIST_IMAGES_OUTPUT}/${CI_IMAGE_PR_TAR}.gz
             gunzip ${CI_DIST_IMAGES_OUTPUT}/${CI_IMAGE_PR_TAR}.gz
-            echo "::set-output name=PR_IMAGE_RESTORED::true"
+            echo "PR_IMAGE_RESTORED=true" >> "$GITHUB_OUTPUT"
```

Comment:
who consumes the GITHUB_OUTPUT file? is this the file which logs the output of all the steps in the CI run?

Reply:
It only logs what you write into it.  The value can be accessed from other steps by accessing `steps.<step name>.outputs.<variable name>`, e.g. `steps.is_pr_image_build_needed.outputs.PR_IMAGE_RESTORED` in this  case.